src/store/modules/job/mutations.ts

```diff
@@ -30,7 +30,7 @@ const mutations: MutationTree <JobState> = {
     [types.JOB_DESCRIPTION_UPDATED] (state, enums) {
         if (enums) {
             enums.forEach((enumId: any) => {
```

Comment:
```suggestion
            enums.forEach((enum: any) => {
```

Reply:
Renamed it enumInfo as enum is a keyword.